setup.sh

```diff
@@ -46,4 +46,4 @@ mkdir -p elf2tab
 cargo install elf2tab --version 0.6.0 --root elf2tab/
 
 # Install python dependencies to factory configure OpenSK (crypto, JTAG lockdown)
-pip3 install --user --upgrade colorama tqdm cryptography fido2
+pip3 install --user --upgrade colorama tqdm cryptography "fido2>=0.9.1"
```

Comment:
If this list is growing, it might make sense to go for
```shell
pip3 install -r requirements.txt
```

and have the packets in a `requirements.txt` doc? We also have i.e. `tools/configure.py`, and the requirements could then indicate more easily which requirement is from where when things get more complicated.

Reply:
That's something we can do in the `develop` branch for the next release. For `stable` I would like to keep it as a small bugfix.

Reply:
Sounds good!